experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -1516,10 +1522,7 @@ class BottomAppBars extends StatelessWidget {
               bottomNavigationBar: BottomAppBar(
                 child: Row(
                   children: <Widget>[
-                    IconButton(
-                      icon: const Icon(Icons.more_vert),
-                      onPressed: () {},
-                    ),
+                    const MenuExample(),
```

Comment:
Nice, but I think this makes more sense as part of the `Dropdown menus` sections (to be renamed "Menus")

Reply:
Just added two more examples in "Dropdown menus" section and renamed the section to "Menus". 

I also tried to add a selected text menu which shows as one of the configurations [here](https://m3.material.io/components/menus/specs#14846245-f9f0-433f-8105-09f0515d86e4), but seems that there's no easy way to use the `contextMenuBuilder` to build a menu with the M3 menu style. Will try to figure it out later.
<img width="457" alt="Screenshot 2023-01-19 at 4 18 39 PM" src="https://user-images.githubusercontent.com/36861262/213590620-7b972bad-ba91-4cd6-beb5-b14d32c6c219.png">